src/app/components/tabs/tabs-demo.component.html

```diff
@@ -11,3 +11,15 @@
     This tab cannot be closed.
   </sky-tab>
 </sky-tabset>
+
+<h3>Tabs example with counter</h3>
```

Comment:
I'd rather just pitch it to Sky and have them instruct me - its their baby so I don't want to make assumptions. I'm ok with doing a little bit of rework if they want it different!